Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM registry.access.redhat.com/ubi8/ruby-27
+FROM registry.access.redhat.com/ubi9/ruby-31
 
 USER root
-RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
-  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs --skip-broken -y shared-mime-info postgresql13 postgresql13-libs \
-  && dnf clean all \
-  && rm -rf /var/cache/yum
+RUN dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs -y shared-mime-info postgresql rubygem-irb rubygem-rdoc \
+    && dnf clean all \
+    && rm -rf /var/cache/yum
+
+# worksround https://bugzilla.redhat.com/show_bug.cgi?id=2221938
+RUN ln -s /usr/share/gems/gems/rdoc-6.4.0/lib/rdoc.rb /usr/share/ruby/ \
+    ln -s /usr/share/gems/gems/rdoc-6.4.0/lib/rdoc /usr/share/ruby/
```

Comment:
Wouldn't be better to add `rdoc` to the Gemfile? I know it's not nice but it's nicer than this, IMO.

Reply:
I think it is worse to put bloat into the project instead of fixing the image. We don't depend on rdoc and it seems wrong to me to include it because of broken ruby distribution.